test/sanitizer_suppressions/tsan

```diff
@@ -30,13 +30,7 @@ race:zmq::*
 race:bitcoin-qt
 
 # deadlock (TODO fix)
-deadlock:CConnman::ForNode
-deadlock:CConnman::GetNodeStats
 deadlock:CChainState::ConnectTip
-deadlock:UpdateTip
-
-# WalletBatch (unidentified deadlock)
-deadlock:WalletBatch
```

Comment:
What are the steps to verify this patch? Note that the deadlocks are intermittent, so there is no way to run the tests and reproduce them or not reproduce them reliably.

Reply:
TSan searches for _potential_ deadlocks, i.e., it verifies consistency of lock ordering in the code, right?

Reply:
https://github.com/google/sanitizers/wiki/ThreadSanitizerDeadlockDetector

Reply:
The lock order graph is generated at runtime and is not deterministic. It depends on the code paths that are executed, and our tests exercise different code paths in each run.

Reply:
> What are the steps to verify this patch?

I don't know (besides tracking code changes since these suppressions were introduced).

Having a smaller set of suppressions is better, though. If this patch appears wrong in the next year, it could be reverted :)